RayLin.md

```diff
@@ -1,33 +1,27 @@
-# 07/25 ~ 07/29
+# 08/01 ~ 08/05
 
 - [Video Control]
+  - Bug 1291013 - Closed-caption button spacing is asymetric
+    - review cancelled, :jaws is overloaded
+    - ask again after flag removed
```

Comment:
I didn't see a new patch. It was canceled due to need-to-rebase. Please find someone else to review it since you would need to uplift it to Fx50.